extensions/lifecycle/conductor/LifecycleConductor.js

```diff
@@ -325,12 +367,15 @@ class LifecycleConductor {
                                 }
 
                                 isTruncated = result.IsTruncated;
-                                nEnqueued += result.Contents.length;
 
                                 result.Contents.forEach(o => {
                                     marker = o.key;
                                     const [canonicalId, bucketName] = marker.split(constants.splitter);
-                                    queue.push({ canonicalId, bucketName });
+
+                                    if (!this._isBlacklisted(canonicalId, bucketName)) {
+                                        nEnqueued += 1;
```

Comment:
I think keeping a count of the skipped ones as well (logging it alongside `nEnqueued`) would be helpful.